spec/classes/redis_spec.rb

```diff
@@ -1329,6 +1336,34 @@ class { 'redis::globals':
         end
 
         it { is_expected.to contain_file(service_file) }
+
+        it do
+          content = <<-END.gsub(%r{^\s+\|}, '')
```

Comment:
I'm not sure we can use `<<~END` with our rubocop config, but that's the modern Ruby way.

Reply:
The tests do not complain, so I guess it works? :)